xml/admin_caasp_block.xml

```diff
@@ -5,13 +5,261 @@
     %entities;
 ]>
 <chapter xmlns="http://docbook.org/ns/docbook"
- xmlns:xi="http://www.w3.org/2001/XInclude"
- xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"
- xml:id="admin-caasp-block-storage">
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"
+  xml:id="admin-caasp-block-storage">
 <!-- ============================================================== -->
  <title>&blockstore;</title>
+ <remark>
+    Content from: https://rook.io/docs/rook/v1.4/ceph-block.html
+    Ignore "Flex Driver" section
+  </remark>
  <para>
-   Content from: https://rook.io/docs/rook/v1.4/ceph-block.html
-   Ignore "Flex Driver" section
+  Block storage allows a single pod to mount storage. This guide shows how to
+  create a simple, multi-tier web application on Kubernetes using persistent
+  volumes enabled by Rook.
  </para>
- </chapter>
+ <sect1 xml:id="rook-block-storage-provision-storage">
+  <title>Provision Storage</title>
+
+  <para>
+   Before Rook can provision storage, a
+   <link xlink:href="https://kubernetes.io/docs/concepts/storage/storage-classes"><literal>StorageClass</literal></link>
+   and
+   <link xlink:href="ceph-pool-crd.md"><literal>CephBlockPool</literal></link>
+   need to be created. This will allow Kubernetes to interoperate with Rook
+   when provisioning persistent volumes.
+  </para>
+
+  <note>
+   <para>
+    This sample requires <emphasis>at least 1 OSD per node</emphasis>, with
+    each OSD located on <emphasis>3 different nodes</emphasis>.
+   </para>
+  </note>
+
+  <para>
+   Each OSD must be located on a different node, because the
+   <link xlink:href="ceph-pool-crd.md#spec"><literal>failureDomain</literal></link>
+   is set to <literal>host</literal> and the <literal>replicated.size</literal>
+   is set to <literal>3</literal>.
+  </para>
+
+  <note>
+   <para>
+    This example uses the CSI driver, which is the preferred driver going
+    forward for K8s 1.13 and newer. Examples are found in the
+    <link xlink:href="https://github.com/rook/rook/tree/%7B%7B%20branchName%20%7D%7D/cluster/examples/kubernetes/ceph/csi/rbd">CSI
+    RBD</link> directory.
+   </para>
+  </note>
+
+  <para>
+   Save this <literal>StorageClass</literal> definition as
+   <literal>storageclass.yaml</literal>:
+  </para>
+
+<screen>
```

Comment:
YAML examples are missing proper indentation. Please follow examples https://github.com/rook/rook/tree/master/cluster/examples/kubernetes/ceph

Also original document has missing ident. in almost every YAML. https://github.com/SUSE/doc-ses/files/5226623/book-storage-rook_color_en.pdf

Without the correct ident the examples cannot by used.

Reply:
Thanks :) we're still working through the edits here. I'll make sure these are adjusted.